provider/keycloak_oidc_identity_provider.go

```diff
@@ -0,0 +1,114 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+)
+
+func resourceKeycloakOidcIdentityProvider() *schema.Resource {
+	oidcSchema := map[string]*schema.Schema{
+		"backchannel_supported": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     true,
+			Description: "Does the external IDP support backchannel logout?",
+		},
+		"use_jwks_url": {
```

Comment:
This switch in the console, when turned on, reveals an input where the user can specify the JWKs url to use.  I think if the ability to configure this switch is exposed, we also need to support the ability to specify the JWKs url.

My suggestion would be to change this attribute to `jwks_url`, which would be an optional string.  If it is provided, we will set `useJwksUrl` to `true` and `jwksUrl` to the string that was provided.  If this is not provided, we will set `useJwksUrl` to `false`.

Alternatively, if you don't want to add this, you can just remove this attribute altogether.  It's okay if you only want to support the functionality you need right now, and it can always be added later if someone else needs it.